man/bootctl.xml

```diff
@@ -102,7 +102,9 @@
       <xi:include href="standard-options.xml" xpointer="version" />
       <varlistentry>
         <term><option>--path</option></term>
-        <listitem><para>Path to the EFI system partition. The default is /boot.</para></listitem>
+        <listitem><para>Path to the EFI system partition (ESP). If not specified the ESP is searched at
```

Comment:
"searched at" sounds wrong. Maybe "If not specified, /efi, /boot, and /boot/efi are checked in turn."